dev/playbooks/inventory-setup/inventory.j2

```diff
@@ -1,6 +1,6 @@
 [testapp]
-{{ lookup('env','COMPOSE_PROJECT_NAME') }}-test_app_ubuntu-[1:2] ansible_connection=docker
-{{ lookup('env','COMPOSE_PROJECT_NAME') }}-test_app_centos-[1:2] ansible_connection=docker
+{{ lookup('env','COMPOSE_PROJECT_NAME') }}_test_app_ubuntu_[1:2] ansible_connection=docker
+{{ lookup('env','COMPOSE_PROJECT_NAME') }}_test_app_centos_[1:2] ansible_connection=docker
```

Comment:
Could you describe the bug you experiencing that these changes resolve? You've updated `inventory.j2` to use `docker-compose` v1 service naming scheme - it actually breaks the dev env for users with `docker-compose` v2.

I don't think we need to support the v1 naming scheme in the dev environment - I think we can expect up-to-date tools.

Reply:
<img width="1786" alt="No_Issue_AfterChanges" src="https://user-images.githubusercontent.com/56577577/167360118-bf3041e4-45c8-4122-937f-5a893072f9a2.png">
<img width="1786" alt="Issue_Without_Changes" src="https://user-images.githubusercontent.com/56577577/167360142-f454c2d6-2548-4d37-8488-b17a65f94412.png">

Please find attached screen shots for reference , whole solution including role and dev works fine with these two lines changes .

Reply:
> Could you describe the bug you experiencing that these changes resolve? You've updated `inventory.j2` to use `docker-compose` v1 service naming scheme - it actually breaks the dev env for users with `docker-compose` v2.
> 
> I don't think we need to support the v1 naming scheme in the dev environment - I think we can expect up-to-date tools.

Thanks John , I have also added inventory-v2.j2 for docker-compose v2 .